language/stdlib/CONTRIBUTING.md

```diff
@@ -54,20 +54,66 @@ The functional testing framework is very convenient, but can't express all of th
 ## Coding conventions
 
 ### Naming
-- Module names should be capitalized and camel case.
-- File names should be lowercase-underscore; e.g., a module named `MyModule` should be stored in a file named `my_module.move`
-- Modules that declare a single resource or struct type should name the type `T`. The reason for this convention is to avoid redundancy: outside of the module, the type will be referred to as `ModuleName::T`.
-- If a module declares multiple resources/structs, the "main" type that clients of the module will use should be called `T` (this is obviously subjective).
-- Function names should be lowercase-underscore
-- Generic type parameter names should be long and descriptive (e.g., `deposit<Token>(t: Token)`). `T` should not be used as a generic type parameter name to avoid confusion with the resource/struct naming convention.
+- **Module names**: are camel case e.g., `LibraAccount`, `Libra`
+- **Type names**: are camel case e.g., `WithdrawalCapability`, `KeyRotationCapability`
+- **Function names**: are lower snake case e.g., `register_currency`
+- **Constant names**: are upper snake case e.g., `TREASURY_COMPLIANCE_ADDRESS`
+- Generic types should be descriptive, or anti-descriptive where appropriate (e.g. `T` for the Vector generic type parameter, `LibraAccount` for the core `LibraAccount` resource, `deposit<Token>(t: Token)` for depositing a token in the `Libra` module). Most of the time the "main" type in a module should be the same name as the module e.g., `Libra::Libra`, `LibraAccount::LibraAccount`.
+- **Builtins** are unqualified and lower snake case e.g., `vector`, `borrow_global`
```

Comment:
Not a big deal, but not sure that this needs to be a part of the style guide given that contributors can't add new builtins...

Reply:
Yea..I wasn't sure whether to add it or not. I'll take it out in the name of being succinct.